data/test/scenarios/wml_tests/ScenarioWML/EffectWML/special_note_individual_unit.cfg

```diff
@@ -1,7 +1,7 @@
 # wmllint: no translatables
 
 #####
-# API(s) being tested: [effect][special_note],[store_unit],unstore_unit]
+# API(s) being tested: [effect][special_note]
```

Comment:
The original list was correct - this test was added as part of fixing #7153, a bug in `[store_unit]`.

Reply:
Then this is a test of [store_unit]. My intention with this change as well as the others you commented on below was to be more clear on what is meant to be tested vs other functionality the test is relying on in order to work.

Here for example, if the bug occurs in [store_unit], then it's relying on [effect][special_note] to give a special note to the unit and [unstore_unit] to put the unit back on the map, but the specific bug it's testing isn't part of either of those APIs.

Reply:
If these really were unit tests by the strict definition of that term then I'd agree, but using WML to test C++ tends to turn tests that touch more than one API. If `[effect][special_note]` wasn't adding a note then this test would notice, so I'm understanding the purpose of choosing only one of the APIs.

It tests the small part of `[store_unit]` that's influenced by `[effect][special_note]`. It doesn't cover the full features of either API.

Reply:
Is the bug related to the `[effect][special_note]` API specifically, or simply that a unit has the special note? The bug, as I understand it, was the latter.

Reply:
Changed to `[store_unit]`.